### DISCUSSION TITLE
Understanding adding-organisations and what happens to the admin email address

### DISCUSSION THREAD

**Discussion by adrigen** [opencollective/opencollective#4621] [General]
We have a nfp community garden. We receive grants & memberships, but also fees for picking up compost from a cafe. 
To enter in the money we receive, from the cafe I have created a new organisation with the cafe name. It also asked for an email address for the admin, under which I entered the cafe owner's name. 

Now I realise that this might be tied to the News system and that they might end up receiving news as a result of this. They didn't ask for news though.. are they going to be able to opt out?

Also if I try to change the associated admin email addresses (I'd rather remove them than spam our contributors) it only shows my name on the team, but it definitely still shows the business owner's name on the front end. 

I'd love to be able to help meaningfully to this project so I really appreciate any feedback about how to approach an issue if I have missed a correct process. 

I was a bit confused about setting up the organisation for a contributor in the beginning... is it wrong for me to do that? Is the only option to plead with someone else to set up their own org?

**Comment by Betree** [comment]
@adrigen Sorry, I'm only seeing your message now. Could you reach out to support@opencollective.com? You'll find faster assistance and answers to your questions there :slightly_smiling_face:

**Comment by adrigen** [comment]
Thanks, I did.
